### PR TITLE
[blog] Statie 4 - how to start a blog

### PR DESCRIPTION
**Pull request by TomasVotruba**

### CODE REVIEW

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    <br><br>
+    Today I will show you, <strong>how to put your first post</strong>.
+'''
+lang: en
+---
+
+
+## Create a Blog Page
+
+This might be the simplest page to show all you posts:
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: all your posts?

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        <a href="/❴$post['relativeUrl']❵/">
+            <h2>❴$post['title']❵</h2>
+        </a>
+    ❴/foreach❵
+❴/block❵
+```
+
+### You already see
+
+- that all posts are in stored in `$posts` variable
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: are in stored in ? nevím jak to myslíš, třeba to máš dobře, ale radši se ptám :-)

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+```twig
+<!-- source/_posts/2017-03-05-my-last-post.md -->
+
+---
+title: "This my Last Post, Ever!"
+---
+
+# Begin by Letting Go
+
+- I was always afraid of writing my feelings. I though everybody would hate me.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: I though**t** everybody...

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+
+Not really a problem anymoogre.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: anymoore

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+I realized, *feelings are like color of skin*.
+**You are born with them and they are part you nature...**
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: part of your nature?

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+    postRoute: my-blog/:year/:title
+```
+
+Would produce url:
+
+```
+my-blog/2017/my-last-post/
+```
+
+Go it? I know you do! **You are smart.**
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: Go**t** it?

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+
+### ProTip #2: Show last 3 on homepage?
+
+- todo: find on tomasvotruba.cz history
+
+
+In next post, I will you some cool post features.
```
Comment (martinahynkova) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: I will **show** you?

File: source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md (hunk 1 above)
Comment (TomasVotruba) [inline source/_posts/2017/2017-03-06-statie-4-how-to-create-the-simplest-blog.md]: Jak to myslíš?

**Pull request merged**